clusterloader2/testing/experimental/storage/pod-startup/config.yaml

```diff
@@ -71,20 +95,32 @@ steps:
       templateFillMap:
         Group: volume-test
         VolSize: {{$VOL_SIZE}}
-        UseCSI: {{$USE_CSI}}
-        CSIStorageClass: {{$CSI_STORAGE_CLASS}}
+        StorageClass: {{$STORAGE_CLASS}}
+        Provisioner: {{$PROVISIONER}}
 {{ end }}
-{{ if $WAIT_FOR_PVS }}
-- name: Waiting for PVs
+{{if $WAIT_FOR_PVS_CREATED }}
+- name: Waiting for volume creation
+  measurements:
+    - Identifier: WaitForPVsToBeCreated
+      Method: WaitForAvailablePVs
+      Params:
+        desiredPVCount: {{$totalVols}}
+        apiVersion: v1
+        provisioner: {{$EXPECTED_PROVISIONER}}
```

Comment:
I think we want to pass labelSelector too (as in other waits).

Reply:
Hmm, I wonder whether `WaitForPVOptions` should even have a `Selector` field. That was added in 21d99e9d0 but I don't see what it is good for.

Reply:
Can we remove it then?

Reply:
I'm not sure - perhaps it has some purpose with other provisioners that I just don't know about. Let's keep it.

Reply:
It seems suspicious to have it set in some "waits" and not in others.

But OK - these tests are marked as experimental anyway for now ...